Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/rtl/ButtonTopLayers_RTL_LayoutTest.java

```diff
@@ -0,0 +1,75 @@
+package org.catrobat.paintroid.test.espresso.rtl;
```

Comment:
see comments in your previous pull request [PR-421](https://github.com/Catrobat/Paintroid/pull/421)
Here you did the reseting of the test language in a nice way with the tearDown() method, you could do the same for [PR-421](https://github.com/Catrobat/Paintroid/pull/421)